base/titer_model.py

```diff
@@ -142,10 +141,19 @@ def filter_strains(titers, strains):
         0
         """
         return {key: value for key, value in titers.iteritems()
-                if key[0] in strains or key[1][0] in strains}
+                if key[0] in strains and key[1][0] in strains}
```

Comment:
@rneher Just to make sure I understand this change: we want both the test and reference strain to be in the request strains subset because the titer model will only use reference strains for which an autologous measure exists. Is that right?